core/controllers/questions_list.py

```diff
@@ -25,8 +25,10 @@
 from core.domain import skill_domain
 from core.domain import skill_fetchers
 
+from typing import Dict, List, TypedDict
 
-def _require_valid_skill_ids(skill_ids):
+
+def _require_valid_skill_ids(skill_ids: List[str]) -> None:
```

Comment:
Do we still need this? Or can we validate these using the schemas?

Reply:
Did some changes, and basically shifted this method to domain_object_validator to reduce repetition in handlers. PTAL!

And yeah we need this to validate every single skill id.